app.rb

```diff
@@ -95,11 +95,23 @@ class InvalidChannelName < StandardError; end
     redirect full_url
   end
 
+  get /(?<channel>\/[\w]+)?\/(?<project>[\w-]+)\/metadata-list/ do
```

Comment:
So the question is how to return that metadata. When returning multiple versions we need to have some form of structure beyond what we have: 

```
"platform": {
  "platform_version": {
    "architecture": {
      {
        "metadata" : { ... } # its current form
      },
      ...
      ...
    }
  }
}

I don't like the idea of returning different formats from one endpoint which is why I created a new endpoint (assuming that we can't change the current endpoint because of backwards compat reasons.